### PR TITLE
Add marketplace_add_item and marketplace_remove_item

### PR DESCRIPTION
**Pull request by elv-zenia**
Add support for adding/removing to/from a marketplace.

Create Marketplace.js with `MarketplaceAddItem()`, `MarketplaceRemoveItem()`, `StorefrontSectionAddItem()`, and `StorefrontSectionRemoveItem()`.

CLI commands:

`marketplace_add_item <marketplace> <object> <price> [forSale]`

`marketplace_remove_item <marketplace> <object>`

`storefront_section_add_item <marketplace> <sku> [section]`

`storefront_section_remove_item <marketplace> <sku> [writeToken]`

### CODE REVIEW

**Comment by elv-zenia** [comment]
Relates to #35

**Comment by elv-zenia** [comment]
Pushed changes for #36 as well.

File: src/Config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const net = "main"; // Set to "main" or "demov3"
```
Comment (elv-serban) [inline src/Config.js:1]: To switch back to main before merge

**Review by elv-serban — APPROVED** [review]
Looks great.
Reminder to switch the config network to "main" before merge

**Comment by elv-zenia** [comment]
@elv-kevin Will you re-review when you get a chance?

**Pull request merged**